Adapter.C

```diff
@@ -110,6 +110,10 @@ bool preciceAdapter::Adapter::configFileRead()
                     }
                     DEBUG(adapterInfo("    connectivity : " + std::to_string(interfaceConfig.meshConnectivity)));
 
+                    // Reset the displacement when defining the interface
```

Comment:
The comment here is a bit confusing, as the resetting does not happen here. Maybe:

```suggestion
                    // When restarting FSI simulations, we need to account for previous accumulated displacement.
                    // We do this by resetting the displacement when defining the interface.
                    // Since this is a feature that may not work as expected, depending on the implementation of the
                    // structure solver used, we allow the user to disable this feature.
```